release/models/optical-transport/openconfig-optical-attenuator.yang

```diff
@@ -186,7 +186,7 @@ module openconfig-optical-attenuator {
     leaf system-derived-target-output-power {
```

Comment:
just curious - would the system derived target belong in "state", or under "config' similar to optical-amplifier target-gain and target-output-power?

Reply:
Since only the system can define it, I put it under state. So a user could view what the system is doing, but in this mode I don't think they should be able to configure it. Please let me know your thoughts though.

Reply:
State only makes sense to me.

Reply:
lgtm as well